File: TraningAPI/View/MenuView/MenuTableViewCell.m
//
//  MenuTableViewCell.m
//  TraningAPI
//
//  Created by 高橋慧 on 2021/06/23.
//  Copyright © 2021 k-takahashi. All rights reserved.
//

#import "MenuTableViewCell.h"

@implementation MenuTableViewCell

+ (NSString *)cellIdentifer {
    return @"MenuTableViewCell";
}
- (void)awakeFromNib {
    [super awakeFromNib];
}
- (void)prepareForReuse {
    [super prepareForReuse];
    self.menuNameLabel.text = nil;
}

- (void)menuLabelConfigure:(NSString *)text {
    self.menuNameLabel.text = text;
}
@end

File: TraningAPI/View/MenuView/MenuTableViewCell.swift
//
//  MenuTableViewCell.swift
//  TraningAPI
//
//  Created by 高橋慧 on 2021/04/29.
//  Copyright © 2021 k-takahashi. All rights reserved.
//

import UIKit

class MenuTableViewCell: UITableViewCell {
    static let className = "MenuTableViewCell"
    @IBOutlet weak var menuNameLabel: UILabel!
    override func awakeFromNib() {
        super.awakeFromNib()
        // Initialization code
        
    }

    override func setSelected(_ selected: Bool, animated: Bool) {
        super.setSelected(selected, animated: animated)

        // Configure the view for the selected state
    }
    override func prepareForReuse() {
        super.prepareForReuse()
        menuNameLabel.text = nil
    }
    
    internal func menuLabelConfigure(text: String){
        menuNameLabel.text = text
    }
    
    
}
